File: examples/decision-python/eligibility.py
def decide_scholarship(applicant: dict) -> dict:
    income = int(applicant.get("annualHouseholdIncome", 0))
    age = int(applicant.get("age", 0))
    has_disability = bool(applicant.get("hasDisability", False))
    eligible = income < 250_000 and age <= 25
    return {
        "outcomeCode": "ELIGIBLE" if eligible else "INELIGIBLE",
        "explanation": (
            f"Python production evaluated income {income} < 250000 and age "
            f"{age} <= 25; disability={has_disability}."
        ),
    }

File: examples/decision-node/src/eligibility.mjs
export function decideScholarship(applicant) {
  const income = Number(applicant.annualHouseholdIncome);
  const age = Number(applicant.age);
  const hasDisability = Boolean(applicant.hasDisability);
  const eligible = income < 250_000 && age <= 25;
  return {
    outcomeCode: eligible ? "ELIGIBLE" : "INELIGIBLE",
    explanation: `Node production evaluated income ${income} < 250000 and age ${age} <= 25; disability=${hasDisability}.`,
  };
}
